tests/console/cpio.pm

```diff
@@ -0,0 +1,67 @@
+#SUSE"s openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Package: cpio
+# Summary: Create and extract archives with cpio, including all the supported archive formats
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Define the archive formats that will be used
+    my @formats = ("bin", "odc", "newc", "crc", "tar", "ustar", "hpbin", "hpodc");
+
+    # Download a folder with some files that will be used for the test purpose
+    my $testdatadir = "/usr/share";
+    assert_script_run "cd $testdatadir";
+    assert_script_run "wget --quiet " . data_url('console/cpio/topack.tar.gz');
+    assert_script_run "tar -xzvf $testdatadir/topack.tar.gz";
+
+    assert_script_run "mkdir /tmp/archive";
+
+    # Copy the files into an archive using the different archive formats
+    foreach my $format (@formats) {
+        assert_script_run "cd $testdatadir/topack && ls | cpio -o -H $format -O /tmp/archive/test_$format 2> /dev/null";
+    }
+
+    assert_script_run "mkdir /tmp/unpacked";
+
+    # Extract the files from an archive to a folder
+    foreach my $format (@formats) {
+        assert_script_run "cd /tmp/unpacked; mkdir $format; cd $format; cpio -i -H $format -I /tmp/archive/test_$format 2> /dev/null";
+    }
+
+    # Check that the initial files used for testing remain the same after the archive/extract process
+    foreach my $format (@formats) {
+        my $result = script_output "diff  $testdatadir/topack /tmp/unpacked/$format 2>&1";
+        if ($result ne "") {
+            assert_script_run "rm -r /tmp/archive /tmp/unpacked";
+            die "An issue occured with format $format";
+        }
+    }
+
+    # Check a cpio regression : bsc#1189463
+    assert_script_run "echo 1234 > filelist";
+    my $res = script_output "cpio -i -d -v -E filelist </dev/null", proceed_on_failure => 1;
+    die if (index($res, "ioctl") == -1);
```

Comment:
Why not record info as an error with the bug already, if we know it's a regression? or die with the bsc?

Reply:
yeap, I just added a record_soft_failure as it is a known issue